test_syndiffix/oneModel.py

```diff
@@ -150,16 +150,19 @@ def getTopFeatures(featuresJob, numFeatures):
     return featuresJob['features'][:numFeatures]
 
 def getMlFeaturesByThreshold(featuresJob, featureThreshold):
+    k = featuresJob['k']
+    if k == 0:
+        print("SUCCESS: (skipped because not enough features)")
+        quit()
     # We always include the top feature
     features = [featuresJob['features'][0]]
-    topScore = featuresJob['cumulativeScore'][0]
+    topScore = featuresJob['cumulativeScore'][k-1]
     if topScore == 0:
-        # Don't expect this, but you never know
-        return features
+        print("SUCCESS: (skipped because zero score)")
+        quit()
```

Comment:
If you skip 2 dim dataset, they will distort scores when compared to other approaches.

Reply:
I never use 2dim ML scores